charter.html

```diff
@@ -236,28 +259,29 @@ <h2 id=coordination>Coordination</h2>
 <a href=https://www.ecma-international.org/>Ecma</a>,
 <a href=https://ietf.org/><abbr
 title="Internet Engineering Task Force">IETF</abbr></a>, and elsewhere,
-and will transition spec proposals to them when they’re ready for the
-standards track. Groups most likely to be close partners are listed
-below, but this group is expected to coordinate with other groups as
-relevant.
+and will transition <a href=#work-items>Work Items</a> to them when
+they’re ready for the standards track. Groups most likely to be close
+partners are listed below, but this group is expected to coordinate with
+other groups as relevant.
 
 <h3 id=w3c-coordination>W3C Groups</h3>
 
 <dl>
 <dt><a href=https://www.w3.org/Privacy/IG/>Privacy Interest Group (PING)</a>
 <dd>This group will coordinate with PING and will take into
-consideration outputs of PING when evaluating proposals.
+consideration outputs of PING when
+evaluating <a href=#proposals>Proposals</a> and Work Items.
 <dt><a href=https://www.w3.org/2011/webappsec/>Web Application Security
 Working Group (WebAppSec)</a>
 <dd>WebAppSec is expected to be the main destination for transitioning
```

Comment:
change to "WebAppSec is expected to be a destination for transitioning" instead of using the word "main".

Reply:
I've updated this wording in d5951a1. LMK if the new text works for you.

Reply:
Looks good!